src/docs/sdk/performance/frames-delay.mdx

```diff
@@ -0,0 +1,145 @@
+frame. When comparing scenarios 1 and 2, they would pick scenario 2 cause it has more slow frames. While scenario
+2 looks worse, the user experience is way worse for scenario 1 because the slow frames last longer.
+
+That's where a new metric called __frames delay__ comes to the rescue. Frames delay represents the delayed frame
+render duration of all frames. The frame delay for one frame is `actual frame duration - expected frame duration`.
+For example, with 60 fps the expected frame duration is 16.67 ms. When a frame takes 20 ms to render,  the frame
+delay is `20ms - 16.67ms = 3.33ms`. 
+
+Let's calculate the frames delay for scenarios 1 - 3.
+
+Scenario | Calculation | Frames Delay
+-- | -- | --
+1 | (500ms - 16.67ms) * 5 | 2416.65ms
+2 | (20ms - 16.67 ms) * 10 | 33.3ms
+3 | 800ms - 16.67ms | 783.33ms
+
+Now, when prioritizing the user-perceived degraded responsiveness by frames delay, we get  1, 3, 2 instead of
+3, 2, 1 when looking at slow and frozen frames.
+
+## Specification
+
+Frames delay represents the delayed frame render duration of all frames. The frame delay for one frame is
+`actual frame duration - expected frame duration`. For example, with 60 fps the expected frame duration is 16.67 ms.
+When a frame takes 20 ms to render,  the frame delay is `20ms - 16.67ms = 3.33ms`. As the expected frame rate can
+change at any time, SDKs must check the expected frame rate for every frame. The frame delay for a span is the sum
+of frame delays for all slow and frozen frames during the start and end time of the span.
+
+SDKs should attach frames delay in seconds to spans via <Link to="/sdk/performance/span-data-conventions/#mobile">span data</Link>.
+For transactions SDKs should send frames delay via the root span with span data.
+While it makes sense to calculate frames delay in milliseconds, __SDKs must send frames delay in seconds__, as SDKs already
+use seconds for most timestamps. For the specification here, we use milliseconds cause they are easier to read.
+
+The specification is written in the [Gherkin syntax](https://cucumber.io/docs/gherkin/reference/).
+
+
+```Gherkin
+Scenario: Slow and frozen frames
+    Given a frame rate of 60 fps
+    And 2 frames with 300 ms    
+    And 1 frame with 900 ms
+    And 10 frames with 16.76 ms
+    When calculating the frames delay
+    Then the frames delay is (300 - 16.67) * 2 + 900 - 16.67 = 1449.99 ms
+
+Scenario: Frame rate changes
+    Given 1 frame with 200 ms with 60 fps
+    And 2 frame with 500 ms with 120 fps
+    And 10 frames with 8.33 ms with 120 fps
+    When calculating the frames delay
+    Then the frames delay is 200 - 16.67 + (500 - 8.33) * 2 = 1166.67 ms
+
+Scenario: Missing delayed frame information
+    """
+    [nf][ ------- ?? ------ ]   |  nf = normal frame,  ?? = No frame information
+    [---- time interval ----]
+    """
+    Given a frame rate of 60 fps
+    And 1 frame with 16.67 ms
+    And 200 ms pass after without any frame being drawn
```

Comment:
when would that happen? does it make sense to provide an example?

Reply:
Thanks for pointing that out. I updated the scenario.